src/views/registrationform.js

```diff
@@ -41,7 +72,7 @@ export default function RegistrationForm() {
 							inputVariant="outlined"
 							format="MM/dd/yyyy"
 							id="date-picker-inline"
-							label="Set appointment date"
+							label="Date"
 							value={firstday}
```

Comment:
![image](https://user-images.githubusercontent.com/59214503/108075161-9f044b80-709c-11eb-91ed-f8736197343a.png)
Pls check again the original date format

Reply:
![image](https://user-images.githubusercontent.com/59214503/108075448-f60a2080-709c-11eb-9583-84cf6a4a3d0d.png)
And this, too

Reply:
I've changed the format of the date. Please check it.